testharness/build.gradle

```diff
@@ -3,7 +3,11 @@ plugins {
 }
 
 dependencies {
-    implementation project(":service")
+    implementation (project(":service")) {
+        exclude group: 'net.minidev', module: 'json-smart'

```

Comment:
I am unsure why this is necessary tbh; when I run a gradle scan without this exclude + explicit json-smart version declaration on L9 gradle is choosing 2.4.8 for some reason. My current hypothesis is that the exclude + explicit version declaration in `service` is being ignored somehow when it's pulled in as a project dependency here, but I haven't been able to find any documentation to that effect.